### PR TITLE
Feature add forms

### PR DESCRIPTION
**Pull request by Bluette1**
# Project Re-former
- [Requirements](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms)
- This project is about 
  - Creating and using forms in Rails
  - Creating View templates in Rails

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to new_user_path
```
Comment (wathigo) [inline app/controllers/users_controller.rb]: **REQUIRED**
* Redirect to `edit_user_path` if user was saved ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_update.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required.
Great work on this project :clap:
Almost all requirements have been met.

Let's fix one issue in comments before proceeding to the next project.

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo.**

**Review by mosesogwo — APPROVED** [review]
**Thank you for making the change requested.:+1: Your project is approved**

You can now merge the PR

**Good luck in your next project:stars:**

### Status: Approved :heavy_check_mark:

**Comment by Bluette1** [comment]
@wathigo  and @mosesogwo: Thank you very much for your feedback!

**Pull request merged**